### PR TITLE
contact us page was added successfully

### PR DESCRIPTION
**Pull request by SanjaiSam**
Contact us page was added successfully for our application with nice styling. I requests the maintainer of this repo to consider this pull request to get the contact us page.

### CODE REVIEW

File: .deepsource.toml
Comment (4darsh-Dev) [inline .deepsource.toml:1]: do not change this file .

File: frontend/src/components/ContactUsPage.jsx
Comment (4darsh-Dev) [inline frontend/src/components/ContactUsPage.jsx:1]: The design could be better, 
See the vercel preview,

**Review by 4darsh-Dev — COMMENTED** [review]
good work @SanjaiSam 
You can do a few additions to get better levels 
1. Add an image or vector on right side and the form on left 
Ref : 
image/vector : freepik
design : https://dribbble.com/tags/contact-us-page

**Comment by panwar8279** [comment]
Please resolve file conflict issue.

**Comment by 4darsh-Dev** [comment]
@SanjaiSam there is no response from your side 
so we're closing the PR

**Pull request closed without merging**